.github/workflows/announcements.yml

```diff
@@ -0,0 +1,42 @@
+name: Fetch announcements from issue
+on: issue_comment
+jobs:
+  issue_commented:
+    # This job only runs for issue comments
+    name: Update Announcements
+    if: |
+      !github.event.issue.pull_request &&
+      github.event.issue.number == 181 &&
+      (github.event.issue.comment.author_association == "OWNER" ||
+      github.event.issue.comment.author_association == "MEMBER" ||
+      github.event.issue.comment.author_association == "CONTRIBUTOR")
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - uses: actions/cache@v2
+        name: configure pip caching
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/python_requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Install Python dependencies
+        env:
+          ISSUE_NUMBER: ${{ github.event.issue.number }}
```

Comment:
We settled on not using this in an environment variable right? We can take this out since it's not being used anymore